react/ui-components/src/hoc/InboxSearchComposer.js

```
@@ -138,7 +146,7 @@ const InboxSearchComposer = ({configs,headerLabel,additionalConfig,onFormValueCh
         var { isLoading, data, revalidate,isFetching,refetch,error } = eval(`Digit.Hooks.${configs.customHookName}(updatedReqCriteria)`);
     }
     else {
-       var { isLoading, data, revalidate,isFetching,error } = Digit.Hooks.useCustomAPIHook(updatedReqCriteria);
+       var { isLoading, data, revalidate,isFetching,error,refetch } = Digit.Hooks.useCustomAPIHook(updatedReqCriteria);
```

Biome (1.9.4)

[error] Shouldn't redeclare 'isLoading'. Consider to delete it or rename it.

'isLoading' is defined here:


(lint/suspicious/noRedeclare)